x-pack/plugins/ml/public/application/explorer/anomaly_timeline.tsx

```diff
@@ -35,6 +35,54 @@ import { ExplorerNoInfluencersFound } from './components/explorer_no_influencers
 import { SwimlaneContainer } from './swimlane_container';
 import { AppStateSelectedCells, OverallSwimlaneData, ViewBySwimLaneData } from './explorer_utils';
 import { NoOverallData } from './components/no_overall_data';
+import { HelpPopover, HelpPopoverButton } from '../components/help_popover/help_popover';
+
+const AnomalyTimelineHelpPopover = () => {
```

Comment:
Shall we move this component to a separate file maybe 🤔 ? Just to keep `anomaly_timeline` smaller

Reply:
Good suggestion!  I split it off in fb501312994466a0581761530b057a620d669e45